android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboard.java

```diff
@@ -80,6 +80,10 @@ final class KMKeyboard extends WebView {
   private final String fontUndefined = "undefined";
   private GestureDetector gestureDetector;
   private static ArrayList<OnKeyboardEventListener> kbEventListeners = null;
+
+  // Facilitates a 'lazy init' - we'll only check the preference when it matters,
+  // rather than at construction time.
```

Comment:
I'm not sure lazy init wins us anything -- we're going to have to check it anyway when the keyboard is displayed.